### DISCUSSION TITLE
UI/UX Designer

### DISCUSSION THREAD

**Discussion by MaryamMori** [WeAreMahsaAmini/Core#120] [Introductions]
Hi, I'm Maryam and I can help with UI/UX. I work with Figma.

**Comment by sinahnyazdi** [comment]
Hi Maryam, great to have you here. can you help with this? #124 
I think we need to use the main slogen (زن زندگی آزادی) in both Farsi and English or better ideas

**Reply by MaryamMori** [reply]
Hi, I'm sorry for the late reply. I have become sick the last few days and I don't think That I can complete this task. it is better to assign it to someone else. I'm sorry that I couldn't inform you sooner.
